uint8 REGION = 0
uint8 POINT_LOCATION = 1
uint8 OBJECT_SELECTOR = 2
uint8 type
geometry_msgs/PointStamped specified_point
geometry_msgs/PointStamped[] region_corners
tangible/SceneObject selected_object
# This assumes the blocks are on a flat surface